tests/core/filtering/test_contract_on_event_filtering.py

```diff
@@ -11,6 +11,34 @@
 
 @flaky(max_runs=3)
 @pytest.mark.parametrize('call_as_instance', (True, False))
+def test_on_filter_using_get_entries_interface(web3,
+                                               sleep_interval,
+                                               emitter,
+                                               Emitter,
+                                               wait_for_transaction,
+                                               emitter_log_topics,
+                                               emitter_event_ids,
+                                               call_as_instance):
+    if call_as_instance:
+        filter = emitter.on('LogNoArguments', {})
+    else:
+        filter = Emitter.on('LogNoArguments', {})
+
+    txn_hash = emitter.transact().logNoArgs(emitter_event_ids.LogNoArguments)
+    wait_for_transaction(web3, txn_hash)
+
+    with Timeout(30) as timeout:
```

Comment:
Not necessary now, but I'd very much like to rework these tests to not require anything asynchronous and to simply use the ethereum-tester backend.